tests/src/test/java/testdatasource/test-document.txt

```diff
@@ -0,0 +1,12 @@
+Testest document
+
+Testest document
+Testest document, Testest document, Testest document
+Testest document
+
+Testest document
+Testest document
+Testest document
+
+
+
```

Comment:
what is this file? does it need to be in repo?

Reply:
We decided to leave the test data and the config properties in the repo to run the tests on master on demo. Thought this will needed as well to run the backend tests.